opendevin/llm/llm.py

```diff
@@ -421,6 +421,16 @@ def async_streaming_completion(self):
     def supports_vision(self):
         return litellm.supports_vision(self.config.model)
 
+    def supports__prompt_caching(self):
```

Comment:
Why not make this a bool property that is only set once during initialization of the llm class?

Reply:
Do you mean attribute

Reply:
That works, too 😃